api/envoy/config/endpoint/v3/endpoint_components.proto

```diff
@@ -141,15 +141,15 @@ message LocalityLbEndpoints {
 
   // The group of endpoints belonging to the locality specified.
   // [#comment:TODO(adisuissa): Once LEDS is implemented this field needs to be
-  // deprecated and replaced by *lb_endpoints_list*.]
+  // deprecated and replaced by *explicit_lb_endpoints_list*.]
   repeated LbEndpoint lb_endpoints = 2;
 
   // [#not-implemented-hide:]
   oneof lb_config {
     // The group of endpoints belonging to the locality.
     // [#comment:TODO(adisuissa): Once LEDS is implemented the *lb_endpoints* field
     // needs to be deprecated.]
-    LbEndpointList lb_endpoints_list = 7;
+    LbEndpointList explicit_lb_endpoints_list = 7;
```

Comment:
Not to get into a huge bikeshedding situation, but we are going to live with this for a while and I'm not sure "explicit" has much meaning. Perhaps `load_balancer_endpoints` if we must name it something else? WDYT?

/wait-any

Reply:
Sure, it's also a good name IMHO.
Let me update the PR.
/wait